recipe/meta.yaml

```
@@ -20,6 +20,7 @@ requirements:
     - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - autoconf  # [unix]
+    - gettext
```

You probably want this and `libintl-devel  # [not linux]` in host.
```suggestion
    - gettext-tools
```

---

@xhochy quick clarification - are you suggesting that `gettext-tools` should also be moved from build to host? Or just changed from `gettext` to `gettext-tools`